content/concepts/kata/translations.md

```diff
@@ -77,21 +81,39 @@ After a review is completed, the reviewer can finalize it with one of following
 
 ### Approving a translation
 
-TBD
+Pending translation can be approved either by kata author, or users with [privilege to coauthor](/references/gamification/privileges/) who are not the publisher of the translation fork being approved, under following conditions:
+
+- translation is older than a week, or
+- kata author has been inactive for over a month.
+
+When translation is approved, new language version is immediately available for other users to solve. There's also a honor reward, but remember that it's granted to the author of the **initial fork**, and not to the author of the fork which got finally accepted!
 
-- who can approve
-- Approver is marked as coauthor
+Translation approver should also go through the kata discourse and mark suggestions posted by the translator as resolved.
+
+**NOTE:** currently, probably because of a bug, a user who _approves_ a translation is marked as **kata contributor**. Users with "contributor" flag seem to be affected by a bug which prevents them from getting any future points they would be rewarded for authoring activities on the kata they contributed to. Be aware that as a result, approving of a translation can cause you won't be eligible for any Honor points you would earn by translating, editing or fixing the kata in the future, unless the bug is fixed.
```

Comment:
```suggestion
**NOTE:** at the time of writing, a user who _approves_ a translation is marked as a **kata contributor**. Users with the "contributor" flag seem to be affected by a bug preventing them from earning any future points they would be rewarded for authoring activities on kata they contributed to. Be aware that as a result, approving a translation will render you ineligible for honor points you would otherwise earn by translating, editing or fixing the kata in the future unless the bug is fixed.
```